_includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-product-offer-feature.md

```diff
@@ -0,0 +1,1822 @@
+            MerchantProductOffersSelectWidget::class,
+        ];
+    }
+}
+```
+
+Enable Javascript and CSS changes:
+
+```bash
+console frontend:yves:build
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following widgets were registered:
+
+| MODULE                            | TEST                                                                                                                               |
+|-----------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| MerchantProductOfferWidget        | Go to a product concrete detail page that has offers, and you will see the default offer is selected, and the widget is displayed. |
+| MerchantProductOffersSelectWidget | Make sure that `ProductConcreteAddWidget` renders product offers list after performing a product search.                           |
+
+{% endinfo_block %}
+
+### 5) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                                | DESCRIPTION                                                                | PREREQUISITES | NAMESPACE                                                              |
+|-------------------------------------------------------|----------------------------------------------------------------------------|---------------|------------------------------------------------------------------------|
+| MerchantProductOfferProductQuickAddFormExpanderPlugin | Expands `ProductQuickAddForm` with `product_offer_reference` hidden field. |               | SprykerShop\Yves\MerchantProductOfferWidget\Plugin\ProductSearchWidget |
+
+**src/Pyz/Yves/ProductSearchWidget/ProductSearchWidgetDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ProductSearchWidget;
+
+use SprykerShop\Yves\MerchantProductOfferWidget\Plugin\ProductSearchWidget\MerchantProductOfferProductQuickAddFormExpanderPlugin;
+use SprykerShop\Yves\ProductSearchWidget\ProductSearchWidgetDependencyProvider as SprykerProductSearchWidgetDependencyProvider;
+
+class ProductSearchWidgetDependencyProvider extends SprykerProductSearchWidgetDependencyProvider
+{
+    /**
+     * @return list<\SprykerShop\Yves\ProductSearchWidgetExtension\Dependency\Plugin\ProductQuickAddFormExpanderPluginInterface>
+     */
+    protected function getProductQuickAddFormExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOfferProductQuickAddFormExpanderPlugin(),
+        ];
+    }
+}
+```
+
+## Related features
+
+| FEATURE                                              | REQUIRED FOR THE CURRENT FEATURE | INTEGRATION GUIDE                                                                                                                                                                                                                                            |
```

Comment:
Review the list of features and make sure that the links are correct. Some of these features were moved to PBC, and their respective IGs are now located in PBC as well.